### PR TITLE
[CARBONDATA-655][CARBONDATA-604]Make no kettle dataload flow as default

### PR DESCRIPTION
**Pull request by ravipesala**
Make no kettle data load flow as default in carbon

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/645/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/656/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/657/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/659/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/660/

File: integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala [hunk 1]
```diff
@@ -390,11 +390,19 @@ case class LoadTable(
       val useKettle = options.get("use_kettle") match {
         case Some(value) => value.toBoolean
         case _ =>
-          val useKettleLocal = System.getProperty("use.kettle")
+          var useKettleLocal = System.getProperty("use.kettle")
+          if (useKettleLocal == null && sqlContext.sparkContext.getConf.contains("use_kettle")) {
```
Comment (jackylk) [inline integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala]: It is strange that "use.kettle", "use_kettle", and "useKettle" are all different, can we use the same name for all  these  configuration?

Comment (chenliang613) [inline integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala]: Same comments as jackly.  i suggest using  "useKettle", what is your opinion? @jackylk

Comment (ravipesala) [inline integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala]: Ok unified.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/670/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/671/

**Comment by ravipesala** [comment]
@jackylk Handled comment, please review

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**